pkgs/tools/video/yamdi/default.nix

```diff
@@ -0,0 +1,36 @@
+{
+  stdenv,
+  fetchurl,
+  ...
+}:
+
+with stdenv.lib;
```

Comment:
Putting this here isn't wrong. This expression however does not use something from `stdenv.lib` anywhere but `meta`. Putting it there might be cleaner/nicer and common throughout `nixpkgs`, like: `meta = with stdenv.lib; {`